src/main/flight/gps_rescue.c

```diff
@@ -511,6 +513,13 @@ void updateGPSRescueState(void)
             // When not in failsafe mode: leave it up to the sanity check setting.
         }
 
+        //set new descent distance if actual distance to home is lower 
+        if (rescueState.sensor.distanceToHomeM < gpsRescueConfig()->descentDistanceM) {
+            descentDistance = rescueState.sensor.distanceToHomeM - 1000;
```

Comment:
What is the `- 1000` in here for?

Reply:
a bit confusing because altitude is espressed in centimeters and distance is espressed in meters, fixed to -5 meters